Documentation and change logs for pronouns settings (#19632)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/organizationsettings.md

| [List microsoftApplicationDataAccessSettings](../api/organizationsettings-list-microsoftapplicationdataaccess.md) | [microsoftApplicationDataAccessSettings](microsoftapplicationdataaccesssettings.md) | Get the properties of a [microsoftApplicationDataAccessSettings](microsoftapplicationdataaccesssettings.md) object that specify access from Microsoft applications to Microsoft 365 user data in an organization. |
| [List peopleInsights](../api/organizationsettings-list-peopleinsights.md) | [insightsSettings](insightssettings.md) | Get the properties of an [insightsSettings](insightssettings.md) object for displaying or returning people insights in an organization. |
| [List profileCardProperties](../api/organizationsettings-list-profilecardproperties.md) | [profileCardProperty](profilecardproperty.md) collection | Get a **profileCardProperty** object collection. |

> | [List pronounsSettings](../api/organizationsettings-list-pronounssettings.md)|[pronounsSettings](../resources/pronounssettings.md) resource|Get the pronounsSettings resources from the pronouns navigation property.|

Reviewer: Line 27:
That description technically just explained syntactically what a GET operation does.
I think more meaningful for developers (who should be familiar with REST GET operations in general) is what this GET operation does for them, what pronounsSettings do for them.

Can you rephrase the description as:
"Get the properties of the pronounsSettings resource for an organization, that controls the display of pronouns in profile cards within the organization."

Please continue to include a link for pronounsSettings.

| [List microsoftApplicationDataAccessSettings](../api/organizationsettings-list-microsoftapplicationdataaccess.md) | [microsoftApplicationDataAccessSettings](microsoftapplicationdataaccesssettings.md) | Get the properties of a [microsoftApplicationDataAccessSettings](microsoftapplicationdataaccesssettings.md) object that specify access from Microsoft applications to Microsoft 365 user data in an organization. |
| [List peopleInsights](../api/organizationsettings-list-peopleinsights.md) | [insightsSettings](insightssettings.md) | Get the properties of an [insightsSettings](insightssettings.md) object for displaying or returning people insights in an organization. |
| [List profileCardProperties](../api/organizationsettings-list-profilecardproperties.md) | [profileCardProperty](profilecardproperty.md) collection | Get a **profileCardProperty** object collection. |

> | [Get pronounsSettings](../api/pronounssettings-get.md)|[pronounsSettings](../resources/pronounssettings.md) collection|Get the pronounsSettings resources from the pronouns navigation property.|

Reviewer 2: is graph convention to call all List ?

|peopleInsights|[insightsSettings](insightssettings.md)| Contains the properties that are configured by an administrator for the visibility of a list of people [relevant and working with](/graph/people-example#including-a-person-as-relevant-or-working-with) a user in Microsoft 365. [List peopleInsights](../api/organizationsettings-list-peopleinsights.md) returns the _settings_ to display or return people insights in an organization.|
|profileCardProperties|[profileCardProperty](profilecardproperty.md) collection| Contains a collection of the properties an administrator has defined as visible on the Microsoft 365 profile card. [Get organization settings](../api/organizationsettings-get.md) returns the properties configured for profile cards for the organization.|

> |pronouns|[pronounsSettings](../resources/pronounssettings.md)|Contains the properties that are configured by an administrator as a tenant-level control whether to display pronouns in an organization.|

Reviewer 2: the tope part of this docs seems missing pronouns

|peopleInsights|[insightsSettings](insightssettings.md)| Contains the properties that are configured by an administrator for the visibility of a list of people [relevant and working with](/graph/people-example#including-a-person-as-relevant-or-working-with) a user in Microsoft 365. [List peopleInsights](../api/organizationsettings-list-peopleinsights.md) returns the _settings_ to display or return people insights in an organization.|
|profileCardProperties|[profileCardProperty](profilecardproperty.md) collection| Contains a collection of the properties an administrator has defined as visible on the Microsoft 365 profile card. [Get organization settings](../api/organizationsettings-get.md) returns the properties configured for profile cards for the organization.|

> |pronouns|[pronounsSettings](../resources/pronounssettings.md)|Contains the properties configured by an administrator as a tenant-level control on whether to display pronouns in an organization.|

Reviewer: Line 43:
There are some repetitions in the description (e.g., "tenant-level", "in an organization"), and "display pronouns in an organization" is too broad a context. We certainly don't mean in other forms of expression within an organization, like in an email body or document, this setting controls whether users can convey their preferred pronouns. 

I think we can reword the description as:
"Represents administrator settings that control the display of pronouns in users' profile cards within an organization."

(There are a couple of other descriptions in this table that can be simplified too.)

Reviewer: Line 43:
Per Anton's email on 1/18,... I suggest rephrasing the description as follows:
"Represents administrator settings that manage the support of pronouns in an organization."

> |pronouns|[pronounsSettings](../resources/pronounssettings.md)|Contains the properties that are configured by an administrator as a tenant-level control whether to display pronouns in an organization.|

Reviewer 3: ```suggestion
|pronouns|[pronounsSettings](../resources/pronounssettings.md)|Contains the properties configured by an administrator as a tenant-level control on whether to display pronouns in an organization.|
```
